### DISCUSSION TITLE
is anychance that OneDriveGUI can have a webui version?

### DISCUSSION THREAD

**Discussion by 1123123w** [bpozdena/OneDriveGUI#87] [Ideas]
I am currently using the Docker version of driveone/onedrive on Unraid. While searching for a web GUI, I came across this project which seems to be good enough. So, I was wondering if it is possible to easily transform this project into a web UI?"

**Comment by bpozdena** [comment]
Nothing has changed since I last answered your question in https://github.com/bpozdena/OneDriveGUI/issues/70 .
